elements/rh-code-block/rh-code-block.ts

```diff
@@ -172,6 +205,7 @@ export class RhCodeBlock extends LitElement {
           </svg>
         </button>
       </div>
+
```

Comment:
The `<button>` element on line 196 needs the `aria-expanded` and `aria-controls` attributes to communicate state to AT. Follow the APG guidelines for [disclosure](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/) to see how to implement.

Reply:
PTAL TY

Reply:
LGTM TY